docs/ref/models/querysets.txt

```diff
@@ -1768,6 +1768,12 @@ This has a number of caveats though:
   does not retrieve and set the primary key attribute, as ``save()`` does.
 * It does not work with many-to-many relationships.
 
+.. versionchanged:: 1.9
+
+    ``bulk_create`` now works with proxy models. Previously it worked for
```

Comment:
Actually, I think "Support for using bulk_create() with proxy models was added." should suffice. I find the "Previously..." sentence a bit awkward and unnecessary.